catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -193,7 +198,9 @@ public QueryResponse federate(List<Source> sources, QueryRequest queryRequest) {
     }
 
     QueryResponse queryCache(QueryRequest queryRequest) {
-        QueryRequest updatedRequest = getRequestWithTagsFilter(queryRequest);
+        QueryRequest updatedRequest = validationQueryFactory.getQueryRequestWithValidationFilter(
```

Comment:
Looks like this is duplicated between the `queryCache` and `sourceFederate`.  I would suggest extracting this logic into a separate method to avoid someone changing one and not the other.

At some point we might consider creating a new kind of plugin that only modifies queries for the local provider but this is ok for now.